docs/user-guide/admin-cli.md

```diff
@@ -653,17 +701,152 @@ enhancement tracked [here](https://github.com/vmware/docker-volume-vsphere/issue
+```
+
+If the Config DB is not initialized, Docker Volume Service will use  "Config DB NotConfigured" Mode, when any request to create, remove, mount or unmount Docker volume is accepted.
+
+After initialization the service can use SingleNode mode - when the DB itself is located on the local ESXi node in `/etc/vmware/vmdkps/auth-db` file, or MultiNode mode - when the above location is a symlin to a shared datastore location.
+
+In SingleNode mode all vm-groups and authorization control is local for each ESXi node, and node do not share this information.
+
+In MultiNode mode, VSphere Docker Volume Service Authorization Config DB needs to be initialized on each ESXi host (`config init --datastore=<ds>`, and the nodes will share the authoration control.
+
+#### Init (`config init`)
+
+Initializing the config is optional. If the config is not initialized, there will be no access control and all `vm-group` commands will fail with appropriate messages.
+
+Before configuring access control or quotas, the config needs to be inited to either SingleNode (`init --local`) or MultiNode (`init --datastore=ds_name`) mode.
+
+```
+[root@localhost:~] vmdkops_admin config init -h
+usage: vmdkops_admin.py config init [-h] [--local] [--force]
+                                    [--datastore DATASTORE]
+
+optional arguments:
+  -h, --help            show this help message and exit
+  --local               Allows local (SingleNode) Init
+  --force               Force operation, ignore warnings
+  --datastore DATASTORE
+                        Config DB will be placed on a shared datastore
+```
+
+Example:
+
+```
+[root@localhost:~] vmdkops_admin status
+Version: 0.12.fea683a-0.0.1
+Status: Running
+DB_LocalPath: /etc/vmware/vmdkops/auth-db
+DB_SharedLocation: n/a
+DB_Mode: NotConfigured (no local DB or symlink)  <===== NOT CONFIGURED
+Pid: 5979199
+Port: 1019
+LogConfigFile: /etc/vmware/vmdkops/log_config.json
+LogFile: /var/log/vmware/vmdk_ops.log
+LogLevel: DEBUG
+
+[root@localhost:~] vmdkops_admin config init --local
+Creating new DB at /etc/vmware/vmdkops/auth-db
+Restarting the vmdkops service to pick up new configuration
+Stopping vmdkops-opsd with PID=5979199
+vmdkops-opsd is not running
+Starting vmdkops-opsd
+vmdkops-opsd is running pid=5979684
+
+[root@localhost:~] vmdkops_admin status
+Version: 0.12.fea683a-0.0.1
+Status: Running
+DB_LocalPath: /etc/vmware/vmdkops/auth-db
+DB_SharedLocation: /etc/vmware/vmdkops/auth-db
+DB_Mode: SingleNode (local DB exists)    <==== LOCAL configuration
+Pid: 5979684
+Port: 1019
+LogConfigFile: /etc/vmware/vmdkops/log_config.json
+LogFile: /var/log/vmware/vmdk_ops.log
+LogLevel: DEBUG
+```
+
+
+#### Remove (`config rm`)
+
+Allows to remove local configuration DB. Since this is a destructive operation, admin needs to type both `--local` flag (to confirm it's local only operation and does not impact shared database, if any) , and `--confirm` flag to confirm that she actually wants to delete the local Config DB.
```

Comment:
OK. Makes sense. I agree it's safes to only support local config removal right now. We can add support for the shared config removal if the requirement really arises in the future.